custom_components/tuya_local/devices/lucci_fan.yaml

```diff
@@ -15,7 +15,7 @@ primary_entity:
         min: 1
         max: 3
 secondary_entities:
-  - entity: light
+  - entity: switch
```

Comment:
Why change to a switch. Is it not a light?

Reply:
This can be switched back if needed, updated to a switch for some of my automations. A switch is also more flexible as you can use a helper to change to a light type